src/modsadmins/percivals.ts

```diff
@@ -0,0 +1,6 @@
+// all in lower case
+export const percivalsArray: string[] = [];
+
+export function isPercival(username: string): boolean {
+  return percivalsArray.includes(username.toLowerCase());
+}
```

Comment:
Please add a comment and specify that this is not the Percival role in Avalon gameplay, but rather a new role on the site who help with moderation.

Reply:
Added one :)